include/nbl/video/IGPUCommandBuffer.h

```diff
@@ -823,6 +863,8 @@ class NBL_API2 IGPUCommandBuffer : public IBackendObject
         // created with IGPUDescriptorSetLayout::SBinding::E_CREATE_FLAGS::ECF_UPDATE_AFTER_BIND_BIT
         // or IGPUDescriptorSetLayout::SBinding::E_CREATE_FLAGS::ECF_UPDATE_UNUSED_WHILE_PENDING_BIT.
         core::unordered_map<const IGPUDescriptorSet*,uint64_t> m_boundDescriptorSetsRecord;
+        const asset::IPipeline<const IGPUPipelineLayout>* m_boundPipeline;
+        asset::E_PIPELINE_BIND_POINT m_boundPipelineBindPoint;
```

Comment:
again, pipeline bind points are separate, they do not disturb each other 
https://github.com/Devsh-Graphics-Programming/Nabla/pull/820/files#r1968449101